<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>In2PermisosUI</class>
 <widget class="QWidget" name="In2PermisosUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pantalla</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_cbTabla">
       <item>
        <property name="text">
         <string>Principal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Maquina</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="m_gb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Permisos</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="m_twTabla">
        <column>
         <property name="text">
          <string>ITEM</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>SuperUsuario</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Mantenimiento</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Administrador</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Printer</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="m_pbGuardado">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbSave">
       <property name="text">
        <string>Guardar</string>
       </property>
       <property name="icon">
        <iconset resource="../../In2Apps/In2SinglePass/SinglePass/shared.qrc">
         <normaloff>:/res/images/floppy13.png</normaloff>:/res/images/floppy13.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbExit">
       <property name="text">
        <string>Salir</string>
       </property>
       <property name="icon">
        <iconset resource="../../In2Apps/In2SinglePass/SinglePass/shared.qrc">
         <normaloff>:/res/images/Ok.png</normaloff>:/res/images/Ok.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../In2Apps/In2SinglePass/SinglePass/shared.qrc"/>
 </resources>
 <connections/>
</ui>
